chain/connector/src/connector/values.rs

```diff
@@ -88,24 +87,13 @@ where
     async fn chain_info(&self) -> Result<ChainInfo, Self::Error> {
         self.check_connection_state()?;
 
-        // TODO: update this once Blokli provides this information
         let info = self.query_cached_chain_info().await?;
 
         Ok(ChainInfo {
             chain_id: info.chain_id as u64,
             hopr_network_name: "dufour".into(),
-            contract_addresses: ContractAddresses {
-                announcements: "0x619eabE23FD0E2291B50a507719aa633fE6069b8".parse()?,
-                channels: "0x693Bac5ce61c720dDC68533991Ceb41199D8F8ae".parse()?,
-                network_registry: "0x582b4b586168621dAf83bEb2AeADb5fb20F8d50d".parse()?,
-                network_registry_proxy: "0x2bc6b78B0aA892e97714F0e3b1c74487f92C5884".parse()?,
-                token: "0xD4fdec44DB9D44B8f2b6d529620f9C0C7066A2c1".parse()?,
-                node_stake_v2_factory: "0x098B275485c406573D042848D66eb9d63fca311C".parse()?,
-                node_safe_registry: "0xe15C24a0910311c83aC78B5930d771089E93077b".parse()?,
-                module_implementation: "0xB7397C218766eBe6A1A634df523A1a7e412e67eA".parse()?,
-                ticket_price_oracle: "0xcA5656Fe6F2d847ACA32cf5f38E51D2054cA1273".parse()?,
-                winning_probability_oracle: "0x7Eb8d762fe794A108e568aD2097562cc5D3A1359".parse()?,
-            },
+            contract_addresses: serde_json::from_str(&info.contract_addresses.0)
+                .map_err(|_| ConnectorError::TypeConversion("contract addresses not a valid JSON".into()))?,
```

Comment:
If using the default ones, the intention is probably to instantiate the `ContractAddresses::default()`, the file should not be relied upon.